diff --git a/node_modules/astro/dist/core/routing/manifest/create.js b/node_modules/astro/dist/core/routing/manifest/create.js
index 518b20b..d9ee575 100644
--- a/node_modules/astro/dist/core/routing/manifest/create.js
+++ b/node_modules/astro/dist/core/routing/manifest/create.js
@@ -85,12 +85,6 @@ function createFileBasedRoutes({ settings, cwd, fsMod }, logger) {
         continue;
       }
       if (!isDir && !validPageExtensions.has(ext) && !validEndpointExtensions.has(ext)) {
-        logger.warn(
-          null,
-          `Unsupported file type ${bold(
-            resolved
-          )} found. Prefix filename with an underscore (\`_\`) to ignore.`
-        );
         continue;
       }
       const segment = isDir ? basename : name;
